numba/runtime/nrt.h

```diff
@@ -5,9 +5,9 @@ All functions described here are threadsafe.
 #include <stdlib.h>
 #include <stdio.h>
 
-/* Debugging facilities */
+/* Debugging facilities - enabled at compile-time */
 /* #undef NDEBUG */
-#ifndef NDEBUG
+#if 0
```

Comment:
Do we want to check a different preprocessor symbol, or are we happy to edit this file when we need to debug NRT? @sklam?